treatment-service/controller/treatment.go

```diff
@@ -126,19 +126,6 @@ func (t TreatmentController) FetchTreatment(w http.ResponseWriter, r *http.Reque
 		return
 	}
 
-	err = t.SchemaService.ValidateSchema(projectId, filterParams.AdditionalProperties)
```

Comment:
No longer needed here, checks are already present further down the request processing funnel.